pkgs/desktops/gnome-3/core/gnome-shell-extensions/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
   doCheck = true;
 
   nativeBuildInputs = [ meson ninja pkgconfig gettext glib ];
-  buildInputs = [ spidermonkey_52 ];
+  buildInputs = [ spidermonkey_60 ];
```

Comment:
Only js52 is supported for running the tests: https://gitlab.gnome.org/GNOME/gnome-shell-extensions/blob/3.30.1/meson.build

Reply:
Ah, good catch.